KarhooUISDK/CommonUI/UITexts.swift

```diff
@@ -14,462 +14,446 @@ public enum UITexts {
-        public static let saveAction = "Text.Booking.PassengerDetails.SaveAction".localized
-        public static let add = "Text.Booking.PassengerDetails.Add".localized
+        public static let title = "kh_uisdk_booking_checkout_passenger".localized
+        public static let subtitle = "kh_uisdk_passengerDetails_subtitle".localized
+        public static let saveAction = "kh_uisdk_save".localized
+        public static let add = "kh_uisdk_passengerDetails_addPassenger".localized
     }
     
     public enum QuoteCategory {
-        public static let electric = "Text.QuoteCategory.Electric".localized
-        public static let mpv = "Text.QuoteCategory.MPV".localized
-        public static let saloon = "Text.QuoteCategory.Saloon".localized
-        public static let exec = "Text.QuoteCategory.Exec".localized
-        public static let executive = "Text.QuoteCategory.Executive".localized
-        public static let moto = "Text.QuoteCategory.Moto".localized
-        public static let motorcycle = "Text.QuoteCategory.Motorcycle".localized
-        public static let taxi = "Text.QuoteCategory.Taxi".localized
+        public static let electric = "kh_uisdk_electric".localized
+        public static let mpv = "kh_uisdk_mpv".localized
+        public static let saloon = "kh_uisdk_vehicle_standard".localized
+        public static let exec = "kh_uisdk_exec".localized
+        public static let executive = "kh_uisdk_filter_executive".localized
+        public static let moto = "kh_uisdk_moto".localized
+        public static let motorcycle = "kh_uisdk_moto".localized
+        public static let taxi = "kh_uisdk_taxi".localized
     }
 
     public enum QuoteFilterCategory {
-        public static let vehicleType = "Text.Quotes.FilterVehicleType".localized
-        public static let vehicleClass = "Text.Quotes.FilterVehicleClass".localized
-        public static let vehicleExtras = "Text.Quotes.FilterVehicleExtras".localized
-        public static let ecoFriendly = "Text.Quotes.FilterEcoFriendly".localized
-        public static let fleetCapabilities = "Text.Quotes.FilterFleetCapabilities".localized
-        public static let quoteTypes = "Text.Quotes.FilterQuoteTypes".localized
-        public static let serviceAgreements = "Text.Quotes.FilterServiceAgreements".localized
+        public static let vehicleType = "kh_uisdk_filter_vehicle_types".localized
+        public static let vehicleClass = "kh_uisdk_filter_vehicle_class".localized
+        public static let vehicleExtras = "kh_uisdk_filter_vehicle_extras".localized
+        public static let ecoFriendly = "kh_uisdk_filter_eco_friendly".localized
+        public static let fleetCapabilities = "kh_uisdk_filter_fleet_capabilities".localized
+        public static let quoteTypes = "kh_uisdk_filter_quote_types".localized
+        public static let serviceAgreements = "kh_uisdk_filter_cancellation_and_waiting_time".localized
```

Comment:
can we change the key name to `kh_uisdk_filter_service_agreements`? I'd keep it more generic than limiting to cancellation and waiting time.